tests/fastboot/not-found-test.js

```diff
@@ -0,0 +1,14 @@
+import { setup, visit } from 'ember-cli-fastboot-testing/test-support';
+import { module, test } from 'qunit';
+
+module('FastBoot | not-found', function (hooks) {
+  setup(hooks);
+
+  test('it returns the correct status code', async function (assert) {
+    const visitHome = await visit('/');
+    assert.equal(visitHome.statusCode, 200);
+
+    const visitFoo = await visit('/foo');
+    assert.equal(visitFoo.statusCode, 404);
```

Comment:
Let me add descriptions to help with maintenance and destructure the returned object to better match the [documentation example](https://embermap.github.io/ember-cli-fastboot-testing/docs/status-code).

```suggestion
    let { statusCode } = await visit('/');

    assert.strictEqual(
      statusCode,
      200,
      'The user sees 200 status code when they visit a URL that exists.'
    );

    ({ statusCode } = await visit('/foo-bar-baz'));

    assert.strictEqual(
      statusCode,
      404,
      'The user sees 404 status code when they visit a URL that does not exist.'
    );
```